### DISCUSSION TITLE
Workload of globaleaks and sites number

### DISCUSSION THREAD

**Discussion by nicolaujoan** [globaleaks/globaleaks-whistleblowing-software#3884] [Q&A]
I was wondering if a single globleaks instance is able to handle many sites with many clients on it or what are its limitations

**Comment by evilaliv3** [comment]
From the point of the computation there are no known limits.
 All just depends on the availability of RAM and disk space.

From the point of view of the RAM 8GB of RAM are enough to run 5000 platforms or more.

**Reply by nicolaujoan** [reply]
And  what about the number of tenants a single platform can handle?

**Reply by evilaliv3** [reply]
My previous answers is referred to the tenants.
We are not aware of limitations and we are continously working on optimizations.

**Reply by nicolaujoan** [reply]
Thanks @evilaliv3 !
